install.sh

```diff
@@ -410,15 +418,15 @@ configure_token() {
   if [ $MONDOO_IS_REGISTERED = true ]; then
     purple_bold "\n* Mondoo was successfully registered."
   else
-    red "\n* Failed to register Mondoo. Please reach out to us via Mondoo Community Discord."
+    red "\n* Failed to register Mondoo. Please ensure your registration token is not expired.  For support, reach out to us via the Mondoo Community Discord."
     exit 1
   fi
 }
 
 configure_macos_token() {
   purple_bold "\n* Register Mondoo with Mondoo Cloud"
-  mkdir -p "$HOME/.config/mondoo/"
-  ${MONDOO_CMD} register --config "$HOME/.config/mondoo/mondoo.yml" --token "$MONDOO_REGISTRATION_TOKEN"
+  sudo_cmd mkdir -p "/etc/opt/mondoo/"
```

Comment:
Is that required? The config should still be local by default for macOS

Reply:
Validated, local .config/ is fine.  Changed.

Reply:
@chris-rock I installed the config into /etc/opt/mondoo/ because that's what the Linux installer does and I wanted consistency.   If you have no reason to do this on Linux let's put in another change to use a local installation.